packages/config/src/layer2s/lightlink.ts

```diff
@@ -0,0 +1,27 @@
+import { upcoming } from './templates/upcoming'
+import { Layer2 } from './types'
+
+export const lightlink: Layer2 = upcoming({
+  id: 'lightlink',
+  display: {
+    name: 'LightLink',
+    slug: 'lightlink',
+    description:
+      'LightLink is an Ethereum Layer 2 blockchain that lets dApps and enterprises offer users instant, gasless transactions.',
+    purposes: ['Universal'],
+    category: 'Optimistic Rollup',
```

Comment:
```suggestion
    category: 'Optimium',
```

since you're planning to use Celestia, this is the proper designation